public/js/index.js

```diff
@@ -9,15 +9,17 @@
 console.info(`rq_ui version ${RQ_PARAMS.VERSION} starting`)
 console.info(`rq_ui config format version ${RQ_PARAMS.CONFIG_FORMAT_VERSION}`)
 
-const keyControl = new KeyControl('#keyControl')
-
 /**
- * A simple helper function that expands on jQuery to allow the WIDGET_NAMESPACE
+ * Extends jQuery. To be called on a jQuery element of class 'widget'.
+ *
+ *  @returns {object} - a complete widget configuration object.
  */
-jQuery.fn.getWidgetData = function () {
+jQuery.fn.getWidgetConfiguration = function () {
   return this.data(RQ_PARAMS.WIDGET_NAMESPACE)
 }
 
+const keyControl = new KeyControl('#keyControl')
```

Comment:
I moved the function above the initialisation of `KeyControl` because `getWidgetConfiguration` is called in that class